src/testRunner/unittests/sys/symlinkWatching.ts

```diff
@@ -12,799 +15,801 @@ import {
     osFlavorToString,
     TestServerHostOsFlavor,
 } from "../helpers/virtualFileSystemWithWatch.js";
-describe("unittests:: sys:: symlinkWatching::", () => {

```

Comment:
You could instead just return at the top before any of the other code runs, if you want the diff to be simpler.

Reply:
Cant return from module at top level

Reply:
Not the module, I mean the `describe` callback, i.e. never call a test function.